### PR TITLE
update the xcm fees page

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the XCM Fees page for the latest spec versions:

Polkadot - v0.9.43
Kusama - v0.9.43
Moonbeam - Runtime 2403

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

https://github.com/moonbeam-foundation/moonbeam-docs-cn/pull/323

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/interoperability/xcm/fees.md [hunk 1]
```diff
@@ -293,105 +282,69 @@ You can take the result of the query and then use it to query the **assetTypeUni
...
+This section covers how XCM fees are estimated for remotely transacting through a signed method.
 
 ### Transact Through Signed Fees {: #transact-through-signed-fees }
 
-The transacting through signed method (multilocation derivative account) consists of four XCM instructions: [`DescendOrigin`](https://github.com/paritytech/xcm-format#descendorigin){target=_blank}, [`WithdrawAsset`](https://github.com/paritytech/xcm-format#withdrawasset){target=_blank}, [`BuyExecution`](https://github.com/paritytech/xcm-format#buyexecution){target=_blank} and [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank}.
-
 When [transacting through the multilocation-derivative account](/builders/interoperability/xcm/xcm-transactor/#xcmtransactor-transact-through-derivative){target=_blank}, the transaction fees are paid by the same account from which the call is dispatched, which is a multilocation-derived account in the destination chain. Consequently, multilocation-derived account must hold the necessary funds to pay for the entire execution. Note that the destination token, in which fees are paid, does not need to be register as an XC-20 in the origin chain.
```
Comment (themacexpert) [inline builders/interoperability/xcm/fees.md:348]:
```suggestion
When [transacting through the multilocation-derivative account](/builders/interoperability/xcm/xcm-transactor/#general-xcm-definitions){target=_blank}, the transaction fees are paid by the same account from which the call is dispatched, which is a multilocation-derived account in the destination chain. Consequently, multilocation-derived account must hold the necessary funds to pay for the entire execution. Note that the destination token, in which fees are paid, does not need to be register as an XC-20 in the origin chain.
```

File: builders/interoperability/xcm/fees.md [hunk 2]
```diff
@@ -293,105 +282,69 @@ You can take the result of the query and then use it to query the **assetTypeUni
...
 
 Consider the following scenario: Alice wants to remotely transact in another chain (Parachain ID 888, in the Moonbase Alpha relay chain ecosystem) from Moonbase Alpha using the transact through signed extrinsic. To estimate the amount of tokens Alice's multilocation-derivative account will need to have to execute the remote call, you need to check the transact information specific to the destination chain. To do so, head to the chain state page of [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbeam.network#/chainstate){target=_blank} and set the following options:
```
Comment (themacexpert) [inline builders/interoperability/xcm/fees.md:350]:
```suggestion
Consider the following scenario: Alice wants to remotely transact in another chain (Parachain ID 888, in the Moonbase Alpha relay chain ecosystem) from Moonbase Alpha using the transact through signed extrinsic. To estimate the amount of tokens Alice's multilocation-derivative account will need to have to execute the remote call, you need to check the transact information specific to the destination chain. To do so, head to the chain state page of [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbase.moonbeam.network#/chainstate){target=_blank} and set the following options:
```

File: builders/interoperability/xcm/fees.md [hunk 3]
```diff
@@ -406,7 +359,7 @@ Consider the following scenario: Alice wants to remotely transact in another cha
...
+From the response, you can see that the `transactExtraWeightSigned` is `{{ networks.moonbase_beta.xcm_message.transact.weight.display }}`. This is the weight needed to execute the four XCM instructions for this remote call in that specific destination chain. Next, you need to find how much the destination chain charges per weight of XCM execution. Normally, you would look into the `UnitsPerSecond` for that particular chain. But in this scenario, no XC-20 tokens are burned. Therefore, `UnitsPerSecond` can be use for reference, but do not ensure that the amount of tokens estimated are correct. To get the `UnitsPerSecond` as a reference value, in the same [Polkadot.js Apps page](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbeam.network#/chainstate){target=_blank}, set the following options:
```
Comment (themacexpert) [inline builders/interoperability/xcm/fees.md:362]:
```suggestion
From the response, you can see that the `transactExtraWeightSigned` is `{{ networks.moonbase_beta.xcm_message.transact.weight.display }}`. This is the weight needed to execute the four XCM instructions for this remote call in that specific destination chain. Next, you need to find how much the destination chain charges per weight of XCM execution. Normally, you would look into the `UnitsPerSecond` for that particular chain. But in this scenario, no XC-20 tokens are burned. Therefore, `UnitsPerSecond` can be use for reference, but do not ensure that the amount of tokens estimated are correct. To get the `UnitsPerSecond` as a reference value, in the same [Polkadot.js Apps page](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbase.moonbeam.network#/chainstate){target=_blank}, set the following options:
```

**Review by themacexpert — APPROVED** [review]
LGTM 👍 
I found 1 more image that needed be updated

**Comment by eshaben** [comment]
@themacexpert Thanks for reviewing and fixing the image!! 🙌

**Pull request merged**